workers/unity/Assets/Playground/Scripts/Workers/UnityClient.cs

```diff
@@ -42,6 +42,9 @@ public override void RegisterSystems()
 
             // Client player commands system
             World.GetOrCreateManager<PlayerCommandsSystem>();
+
+            // Metric sending system
+            World.GetOrCreateManager<MetricSendSystem>();
```

Comment:
We will probably want the system on `UnityGameLogic` too :)

Reply:
Done now!